Source/WebKit/UIProcess/Launcher/cocoa/ExtensionProcess.mm

```diff
@@ -83,6 +122,7 @@
     }, [&] (RetainPtr<BENetworkingProcess> process) {
     }, [&] (RetainPtr<BERenderingProcess> process) {
         interaction = [process createVisibilityPropagationInteraction];
+    }, [&] (RetainPtr<_SEExtensionProcess> process) {
```

Comment:
Could use `auto&` here and drop the `RetainPtr<BENetworkingProcess> process` lambda above.